zipline/assets/assets.py

```diff
@@ -288,6 +290,30 @@ def retrieve_equities(self, sids):
         """
         return self._retrieve_assets(sids, self.equities, Equity)
 
+    def _make_info(attr):
```

Comment:
yeah, when I make a function for generating methods I like to keep it next to the use and del it later. I use this pattern in the fixtures for making a lot of "return None" functions. This is harder to understand when it is not in the class imho.